translations/en-us.yaml

```diff
@@ -169,6 +169,13 @@ realmNames:
   euWest: EU West
   euEast: EU East
 
+
+## Error page for UI access removed ##
+legacyRemoved:
+  title: The Legacy UI has been removed
+  message: Please use the new Rancher Dashboard UI
+  action: Goto Dashboard
```

Comment:
typo here

Reply:
@mantis-toboggan-md Help! What's the typo?

Reply:
Lol sorry about that @nwmac - should be
`action: Go to Dashboard`

Reply:
Ah - you want a space! I was writing for BASIC developers.